google/cloud/bigquery/_versions_helpers.py

```diff
@@ -24,6 +24,8 @@
 _MIN_PYARROW_VERSION = packaging.version.Version("3.0.0")
 _MIN_BQ_STORAGE_VERSION = packaging.version.Version("2.0.0")
 _BQ_STORAGE_OPTIONAL_READ_SESSION_VERSION = packaging.version.Version("2.6.0")
+_MIN_PANDAS_VERSION = packaging.version.Version("1.1.0")
+_MAX_PANDAS_VERSION = packaging.version.Version("2.0.0")
```

Comment:
I would prefer we don't enforce this. It will break BigQuery DataFrames, as we use pandas 2.0+ with this library just fine.